### PR TITLE
Bugfix FXIOS-6918 [v120] Create the bottom toolbar UI component

### PR DESCRIPTION
**Pull request by yoanarios**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-6918)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/15396)

## :bulb: Description
- Added cases for Remote tabs
- Use isRegularLayout computed var to update layout to use mock on unit test
- Added unit test

## :pencil: Checklist
You have to check all boxes before merging
- [x] Filled in the above information (tickets numbers and description of your work)
- [x] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [x] Wrote unit tests and/or ensured the tests suite is passing
- [x] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [x] If needed I updated documentation / comments for complex code and public methods

### CODE REVIEW

**Review by mattreaganmozilla — APPROVED** [review]
Still getting familiar with some of this code but nothing major jumped out. Lgtm

File: Client/Frontend/Browser/Tabs/TabTrayViewController.swift [hunk 1]
```diff
@@ -215,5 +342,12 @@ class TabTrayViewController: UIViewController,
     func newTabButtonTapped() {}
 
     @objc
-    func doneButtonTapped() {}
+    func doneButtonTapped() {
+        notificationCenter.post(name: .TabsTrayDidClose)
+        // TODO: Update mode when closing tabTray
```
Comment (OrlaM) [inline Client/Frontend/Browser/Tabs/TabTrayViewController.swift:347]: nit: TODO should have a ticket number

**Review by OrlaM — APPROVED** [review]
One small nit, otherwise looks good.

**Comment by yoanarios** [comment]
Investigating why `testCountUpdatesAfterTabRemoval` is crashing on Bitrise

**Pull request merged**